at_client/lib/src/manager/monitor.dart

```diff
@@ -214,7 +217,7 @@ class Monitor {
   /// Stops the monitor. Call [Monitor#start] to start it again.
   void stop() {
     status = MonitorStatus.Stopped;
-    _monitorConnection.close();
+    _monitorConnection.getSocket().destroy();
```

Comment:
connection.close() internally calls socket.destroy..so this change is not required

Reply:
Reverted with monitor.close() method.